luzer/tests/CMakeLists.txt

```diff
@@ -20,15 +20,55 @@ set_tests_properties(luzer_e2e_test PROPERTIES
 )
 
 add_test(
-  NAME luzer_options_test
+  NAME luzer_options_table_flag_corpus_test
```

Comment:
Why do you rename it?

Reply:
I added a few more tests to check the startup options. Now there are several such tests, so I renamed this test to a more accurate name.

Reply:
okay